src/appleseed/foundation/platform/compiler.h

```diff
@@ -223,6 +223,18 @@ namespace foundation
 #endif
 
 
+//
+//  noexcept support.
+//
+
+// Visual C++ 2012 and earlier.
+#if !defined(_MSC_VER) || _MSC_VER >= 1800
+    #define APPLESEED_NOEXCEPT
```

Comment:
Incorrect, it's the other way around.

Here's a correct implementation, you can copy/paste it:

    // Visual C++ 2012 and earlier: ignore.
    #if defined _MSC_VER && _MSC_VER < 1800
        #define APPLESEED_NOEXCEPT

    // Other compilers.
    #else
        #define APPLESEED_NOEXCEPT noexcept
    #endif

Reply:
Fixed